src/stories/containers/users/create-account/create-account.tsx

```diff
@@ -1,17 +1,23 @@
 import styled from '@emotion/styled';
-import React from 'react';
+import React, { useCallback } from 'react';
 import Image from 'next/image';
 import CloseButton from '../../../components/close-button/close-button';
 import { useThemeContext } from '../../../../core/context/ThemeContext';
 import { CustomButton } from '../../../components/custom-button/custom-button';
 import TextInput from '../../../components/text-input/text-input';
 import { ButtonWrapper, Container, Wrapper, Form } from '../../auth/login/login';
 import { useCreateAccountMvvm } from './create-account.mvvm';
+import { useRouter } from 'next/router';
 
 export default () => {
   const { isLight } = useThemeContext();
+  const router = useRouter();
   const { form, loading, error } = useCreateAccountMvvm();
 
+  const handleGoBack = useCallback(() => {
+    router.back();
```

Comment:
The same comment of the previous `handleGoBack` function applies here. Maybe the functionality can be reused in both places and any other place where it can be useful

Reply:
You are right I will make the change